### PR TITLE
docs: use `token` consistently throughout post

### PR DESCRIPTION
**Pull request by suruaku**
One variable name was missing in some places and it was incorrect.

### CODE REVIEW

**Comment by suruaku** [comment]
Forgot to mention, authtoken name is already used in that example [last code part](https://nuxtjs.org/blog/creating-a-nuxt-module#full-code-example)

Also env variable won't be used since it is assigned to `const token`, but never used? (lines 232 & 238)

File: content/en/blog/creating-a-nuxt-module.md [hunk 1]
```diff
@@ -219,7 +219,7 @@ export default function () {
 }
 ```
 
-We can then await the ngrok authtoken passing in the value of `token`which we have defined above.
+We can then await the ngrok authtoken passing in the value of `authtoken`which we have defined above.
```
Comment (clarkdo) [inline content/en/blog/creating-a-nuxt-module.md:222]: Let's add a space after `

**Review by danielroe — COMMENTED** [review]
@suruaku Very good catch on quite a lot of this. My only suggestion is to use `token` for the variable name (as before) to distinguish it from the ngrok method - but to keep the rest of your changes.

**Comment by suruaku** [comment]
Yes sure, using `token` makes more sense after reading this article for many times.

**Review by danielroe — APPROVED** [review]

**Pull request merged**